### DISCUSSION TITLE
寻找主页

### DISCUSSION THREAD

**Discussion by xiaoj746** [PanJiaChen/vue-element-admin#4206] [General]
我想请问一下，那个主页的代码是在哪个文件下呢

**Comment by lzglocal** [comment]
npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()
npm ERR! Error while executing:
npm ERR! D:\git\cmd\git.EXE ls-remote -h -t git://github.com/adobe-webplatform/eve.git
npm ERR!
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 20.205.243.166]: errno=Unknown error
npm ERR!
npm ERR!
npm ERR! exited with error code: 128
